package.json

```diff
@@ -12,18 +12,19 @@
     "eslint-plugin-prettier": "^4.0.0",
     "jest": "^27.3.1",
     "microbundle": "^0.14.2",
-    "prettier": "^2.5.1"
+    "prettier": "^2.5.1",
+    "typescript-transform-paths": "^3.3.1"
   },
   "source": "src/index.ts",
-  "exports": {
-    "require": "./dist/src/cow-sdk.js",
-    "default": "./dist/src/cow-sdk.modern.js"
-  },
-  "main": "./dist/src/cow-sdk.js",
-  "module": "./dist/src/cow-sdk.module.js",
-  "unpkg": "./dist/src/cow-sdk.umd.js",
+  "main": "dist/index.modern.js",           
+  "types": "dist/index.d.ts",
+  "sideEffects": false,
```

Comment:
is this sideEffects something related to webpack? microbundle uses rollup underneath so is this really necessary?